### PR TITLE
Reworked cms helper to use blockable blocks caching

### PR DESCRIPTION
**Pull request by AndreasSliwka**

### CODE REVIEW

File: app/helpers/comfy/cms_helper.rb [hunk 1]
```diff
@@ -60,14 +62,14 @@ def cms_block_content(identifier, blockable = @cms_page)
...
     # Example: {{cms:field}} tags
     def cms_block_content_render(identifier, blockable = @cms_page)
-      tag = blockable && (block = blockable.blocks.find_by_identifier(identifier)) && block.tag
+      tag = cms_tag_from_identifier_and_blockable(identifier, blockable)
       return '' unless tag
       render :inline => ComfortableMexicanSofa::Tag.process_content(blockable, tag.content)
     end
 
     # Same as cms_block_content but with cms tags expanded
     def cms_block_render(identifier, blockable = @cms_page)
```
Comment (milgner) [inline app/helpers/comfy/cms_helper.rb]: Since `cms_block_render` and `cms_block_content_render` are the same, can't we get rid of the duplication while we're at it? Should be a simple `s/cms_block_content_render/cms_block_render/` as I understand it.

File: lib/comfortable_mexican_sofa/fixture/file.rb [hunk 2]
```diff
@@ -59,6 +46,30 @@ def import!
       # cleaning up
       self.site.files.where('id NOT IN (?) AND block_id IS NULL', fixture_ids).each{ |s| s.destroy }
     end
+
+    protected
+
+    def find_block(attrs, filename)
+      return unless attrs.has_key? 'page'
+      page = self.site.pages.find_by_full_path(attrs['page']) ||
```
Comment (milgner) [inline lib/comfortable_mexican_sofa/fixture/file.rb]: Would it make sense to also use `find` with a block here? It's basically the same situation as above, requiring multiple database queries.

File: test/helpers/cms_helper_test.rb [hunk 3]
```diff
@@ -67,6 +68,8 @@ def test_cms_block_with_files
       ]
     )
 
+    @cms_page.reload
```
Comment (milgner) [inline test/helpers/cms_helper_test.rb]: Why is this required now?

Comment (AndreasSliwka) [inline test/helpers/cms_helper_test.rb]: at this point page.blocks had been preloaded, so we have to clear the cached blocks.
